### ISSUE TITLE
telephone link support

### ISSUE THREAD

**Issue by cx4** [0E800/flarum-ext-bbcode-alerts#2]
The latest version of the support tel link call dial phone number?
like this:
<a href="tel:1-555-555-1212">555-555-1212</a>

**Comment by cx4** [comment]
> `<a href="tel:1-555-555-1212">555-555-1212</a>`

**Comment by 0E800** [comment]
Ill see what I can do. I am not that proficient with including javascript with the Flarum plugins.
What is the application that is typically opened to make the phone call when clicked on?

**Comment by cx4** [comment]
Yes, when you open the flarum on mobile devices, in the article include the phone number, if you want to contact each other, click dial panel of phone Numbers can wake up on a mobile device.
